PWGHF/TableProducer/candidateCreatorXicToXiPiPi.cxx

```diff
@@ -741,7 +943,7 @@ struct HfCandidateCreatorXicToXiPiPiExpressions {
           }
           if (indexRec > -1) {
             RecoDecay::getDaughters(mcParticles.rawIteratorAt(indexRecXicPlus), &arrDaughIndex, std::array{0}, 1);
-            if (arrDaughIndex.size() == 2) {
+            if (arrDaughIndex.size() == arrXiResonance.size()) {
```

Comment:
It is not needed to call `size` every time since the array size is a compile time constant which can be declared as
```c++
constexpr std::size_t NDaughtersResonant{2u};
```
and used in the declaration of `arrXiResonance`.